packages/react/src/components/Pagination/Pagination.tsx

```diff
@@ -16,6 +16,124 @@ import { useFallbackId } from '../../internal/useId';
 import { usePrefix } from '../../internal/usePrefix';
 import { IconButton } from '../IconButton';
 
+type ExcludedAttributes = 'id' | 'onChange';
+
+export interface PaginationPageSize {
+  text: string;
+  value: string;
```

Comment:
Completely appreciate this is still a draft, but just to flag in the case it's helpful, based on this (and its display in the docs);
https://github.com/carbon-design-system/carbon/blob/d30eb7ca032d58bfe631242ffbe7a4df3d23e03f/packages/react/src/components/Pagination/Pagination.tsx#L508-L516

This might need to be a `number`

```suggestion
  value: number;
```